plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/AdvancedConnectionEditor.java

```diff
@@ -343,9 +349,17 @@ public boolean isViewFor(Object object) {
 		return object instanceof ConnectionFactory;
 	}
 
-	private Connection getConnection() {
+	Map<String, Object> map = null;
```

Comment:
@robstryker and then there's a page model already, what's the reasoning to add another model within the UI class where so far special care was applied separate model(-logic) from UI? Why not having pageModel#getExtendedProperties?